debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/LogMinerHelperIT.java

```diff
@@ -0,0 +1,229 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.oracle;
+
+import static org.fest.assertions.Assertions.assertThat;
+
+import java.math.BigDecimal;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TestRule;
+
+import io.debezium.connector.oracle.junit.SkipTestDependingOnAdapterNameRule;
+import io.debezium.connector.oracle.junit.SkipWhenAdapterNameIsNot;
+import io.debezium.connector.oracle.logminer.LogMinerHelper;
+import io.debezium.connector.oracle.logminer.SqlUtils;
+import io.debezium.connector.oracle.util.TestHelper;
+import io.debezium.embedded.AbstractConnectorTest;
+import io.debezium.util.Testing;
+
+/**
+ * This subclasses common OracleConnectorIT for LogMiner adaptor
+ */
+@SkipWhenAdapterNameIsNot(value = SkipWhenAdapterNameIsNot.AdapterName.LOGMINER, reason = "LogMiner specific tests")
+@Ignore(value = "The super class before method fails, needs investigation - alter session insufficient privileges")
+public class LogMinerHelperIT extends AbstractConnectorTest {
```

Comment:
@AndreyIg As shown here, I've marked this test as `@Ignore`.  The problem with this test is I'm getting some insufficient privileges when the `@BeforeClass` handler is called and executes the `#resetSessionToCdb`.  I need to investigate why this is problematic here.

Reply:
Please share your findings.